### PR TITLE
2022 Report

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

File: source/2022.tex [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+            \subsubsection{Injury to Members, Conference Attendees or Trustees}
+            The UKPA requires insurance to cover this potential risk and the trustees intend to put that insurance in place once the UKPA has sufficient funds to cover the cost of the premium.
+
+            \subsubsection{Reduction or Loss of Volunteers}
+            The UKPA is totally reliant upon volunteers to run and administer the activities of the UKPA. If there were a reduction in the number of volunteers to an unacceptable level, then there would have to be a contraction, consolidation or closure of activities.
+
+    \section{Achievements and Performance}
+    	As a result of the ongoing COVID19 pandemic, the trustees decided to cancel the 2021 PyCon UK event and there was no other significant activity during the course of the year.
```
Comment (doismellburning) [inline source/2022.tex]: nitpick / suggestion: Most style guides seem to use `COVID-19`; I've not really seen `COVID19` in any usage

File: source/2022.tex [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    \section{Financial Review}
+
+        \subsection{Reserves Policy}
+		The UKPA has undertaken a review of its potential liabilities in the face of foreseeable risks and determined that a reasonable sum to hold in reserve is that required to cover a year's operational expenditure plus some contingency for unforeseen costs: \pounds10K
+
+        At the end of March 2022, the UKPA held reserves of \pounds TODO.
+
+        All funds raised from previous PyCon UK conferences are currently held by PyCon UK Society Ltd (PUKSL). The intention is that these funds will be transferred to the new subsidiary trading company  and PUKSL will be closed.
```
Comment (doismellburning) [inline source/2022.tex]: typo: `company  and` (double space)

**Review by doismellburning — APPROVED** [review]
Brilliant, thank you. Two minor things in-line but they're both very minor - thank you for putting this together. Will PR in the numbers later.

**Review by drvinceknight — APPROVED** [review]

**Comment by AnnJBarr** [comment]
LGTM

**Review by AnnJBarr — APPROVED** [review]

**Pull request merged**